process/sig-lifecycle.md

```diff
@@ -1,13 +1,30 @@
 # Special Interest Group (SIG) Life Cycle
 
-Special Interest Groups are under the direct governance of a Working Group (WG) which can create and close them as necessary and appropriate. SIGs are expected to have a very specific goals and objectives at the time of their creation which may or may not define if they are terminable and is entirely dependent on the nature of the SIG.
+Special Interest Groups are under the direct governance of their reporting Working Group (WG) and are bound to achieving a very specific goal. These groups may be terminated upon completion of their designating tasking, continued for larger and ongoing efforts, or otherwise subject to the governance, structure, and termination policies of the WG they are under. The creation of a SIG must dictate the focus, intent, goals, and deliverables as appropriate.
```

Comment:
This language came from https://github.com/ossf/tac/blob/main/organizational-structure-overview.md

Reply:
https://github.com/ossf/Diagrammers-Society today is a SIG that reports to the TAC. An explicit goal of this iteration is not to undo past decisions, but this would prevent new SIGs from forming under the TAC.

Reply:
I think we should allow SIGs to report directly to the TAC and update everything accordingly.
There is no harm in doing this and I don't know that the restriction to have SIGs only report to a WG was intentional in the first place. I think it was simply what we had at the time.

Reply:
I have updated the text!